keyvault/client/keys/src/main/java/com/azure/keyvault/keys/models/webkey/Base64UrlJsonDeserializer.java

```diff
@@ -24,7 +24,7 @@
         if (text != null) {
             return BASE64.decode(text);
         }
-        return null;
+        return new byte[0];
```

Comment:
Discuss with @g2vinay - this might be suppressed.

Reply:
Same with other code that is similar below...

Reply:
It was not suppressed earlier, I have added an exclusion rule for this and reverted this change to return `null` instead in both places.